ui/src/onboarding/actions/index.ts

```diff
@@ -62,7 +64,7 @@ export const setBucketID = (bucketID: string): SetBucketID => ({
   payload: {bucketID},
 })
 
-export const setupAdmin = (params: ISetupParams) => async dispatch => {
+export const setupAdmin = (params: ISetupParams) => async (dispatch): Promise<boolean | null> => {
```

Comment:
Each type in typescript also contains null as a subtype, so you can just say boolean here and null should be accepted

Reply:
Ok, I fixed it to be boolean type